### PR TITLE
Add Deployment Example

### PR DESCRIPTION
**Pull request by adrianchiris**
Add some examples on how to:

1. Persistent netdev name via udev rule
2. Deploy network-operator
3. Depoloy secondary Network: multus + macvlan + whereabouts IPAM
4. Deploy Workloads - RDMA and GPUDirect RDMA

Example should be slightly modified to fit each and every deployment
but provides a good starting point to getting the components deployed.

### CODE REVIEW

**Comment by adrianchiris** [comment]
@moshe010  Should we have an example for a basic deployment of the operator ?
if yes, then i have compiled something here.

This is regardless of documentation that should be added in general for the project.

File: example/delete-operator.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (moshe010) [inline example/delete-operator.sh:1]: copyright for bash?

File: example/rdma-test-pod2.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: ubuntu00
+  restartPolicy: OnFailure
+  containers:
+  - image: mellanox/rping-test 
+    name: rdma-test-ctr
+    securityContext:
+      capabilities:
+        add: [ "IPC_LOCK","NET_ADMIN" ]
```
Comment (moshe010) [inline example/rdma-test-pod2.yml:16]: why NET_ADMIN?

File: example/udev/00_persistent_netdev [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# Persistent netdev example using PCI
```
Comment (moshe010) [inline example/udev/00_persistent_netdev:1]: why this is needed?

Comment (adrianchiris) [reply example/udev/00_persistent_netdev:1]: its an example of how to do persistent dev name,
you need the same netdev name across setups for rdma device plugin and macvlan cni

i can add a comment explaining that in the file

File: example/rdma-test-pod2.yml (hunk 2 above)
Comment (adrianchiris) [reply example/rdma-test-pod2.yml:16]: if you are not using a cluster IPAM on secondary network and are assigning manually.

Today we dont have an official solution for secondary network IPAM what is cluster unique right ?

there are two net-attach-def CRs in the example, one with whareabouts IPAM and one without.

File: example/delete-operator.sh (hunk 1 above)
Comment (adrianchiris) [reply example/delete-operator.sh:1]: yes!

File: example/rdma-test-pod2.yml (hunk 2 above)
Comment (moshe010) [reply example/rdma-test-pod2.yml]: why not static cni? https://github.com/containernetworking/plugins/tree/master/plugins/ipam/static
You can assign with it in the pod spec. Why we need to elevate the pod with  NET_ADMIN just to put static ip address?

File: example/udev/00_persistent_netdev (hunk 3 above)
Comment (moshe010) [reply example/udev/00_persistent_netdev]: why the operator is not doing it by default?

Comment (adrianchiris) [reply example/udev/00_persistent_netdev]: This was not in the scope, we should discuss more.
At machine boot you need to have some sort of constant that is the same across machines, would that be the PCI address ?

File: example/rdma-test-pod2.yml (hunk 2 above)
Comment (adrianchiris) [reply example/rdma-test-pod2.yml]: i was not aware of this CNI.
This is a better alternative than configuring within the pod which requires additional caps.

will modify

**Pull request merged**